src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,50 @@
+package duke.storage;
+
+import duke.exceptions.DukeException;
+import duke.dataprocessing.Decoder;
+import duke.tasks.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    String filepath;
+
+    public Storage(String filepath) throws DukeException {
+        this.filepath = filepath;
+    }
+
+    public List<Task> load() throws DukeException {
+        List<Task> taskList = new ArrayList<>();
+        File f = new File(filepath);
+        Scanner s = null;
+        try {
+            s = new Scanner(f);
+        } catch (FileNotFoundException e) {
+            throw new DukeException("OOPS!!! File is not found.");
+        }
+        while (s.hasNext()) {
+            Task decoded = Decoder.decodeTask(s.nextLine());
+            taskList.add(decoded);
+        }
+        return taskList;
+    }
+
+    public void update(TaskList taskList) throws DukeException {
```

Comment:
A better name can be 'updateToFile'

Reply:
Agree. Perhaps a more specific method name is better?